src/mobile/pages/pages-test.js

```diff
@@ -474,33 +476,112 @@ describe('<a2j-pages>', () => {
   })
 
   describe('Component', () => {
-    let appStateTeardown
```

Comment:
I moved the lines here into its own `describe` so they can be updated/fixed later.